### PR TITLE
Add declarations for "plotly.js/lib/core"

### PR DESCRIPTION
**Pull request by gnestor**
This resolves an issue with plotly.js + typescript + webpack: https://github.com/jupyterlab/jupyter-renderers/issues/10

### CODE REVIEW

**Comment by gnestor** [comment]
@frederikaalund The issue is that when using plotly.js with webpack, the default export causes a runtime error (see https://github.com/jupyterlab/jupyter-renderers/issues/10#issue-245487543 for details). Importing `plotly.js/lib/core` resolves this issue, but the type declarations provided by `@types/plotly.js` don't provide declarations for the `plotly.js/lib/core` module (even though that module is identical to the default module). 

My solution: Add a type declaration at "plotly.js/lib/core" and simply export everything from the default export. Is there a better way to do this?

**Comment by blink1073** [comment]
This looks good to me.  @gnestor, the renderer works with this change locally to `node_modules`.

**Comment by gnestor** [comment]
@typescript-bot Fixed!

**Comment by gnestor** [comment]
@RyanCavanaugh This is ready to be reviewed

File: types/plotly.js/tsconfig.json [hunk 1]
```diff
@@ -18,6 +18,8 @@
     },
     "files": [
         "index.d.ts",
-        "plotly.js-tests.ts"
+        "lib/core.d.ts",
```
Comment (sheetalkamat) [inline types/plotly.js/tsconfig.json]: Please do not copy paste the file to create duplicate typings. 

You would want to follow something like what d3 typings do.
Please take a look at: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3/index.d.ts and the corresponding sub typings you can get eg. https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-array/index.d.ts

This is what you need to do here as well.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
This isnt the right change, please take a look at the comment for more details

File: types/plotly.js/tsconfig.json (hunk 1 above)
Comment (gnestor) [inline types/plotly.js/tsconfig.json]: Hi @sheetalkamat, thanks for taking a look! If you look at `lib/core.d.ts`, it is simply doing an `export * from 'plotly.js';`. The copied-and-pasted code is the test, which can be removed. The goal with this PR is to allow users to import from `plotly.js/lib/core` in addition to plain old `plotly.js` while still importing type declarations.

**Review by sheetalkamat — APPROVED** [review]

**Review by sheetalkamat — APPROVED** [review]

**Comment by gnestor** [comment]
Thanks @sheetalkamat!

**Pull request merged**